src/find/mod.rs

```diff
@@ -171,6 +171,7 @@ Early alpha implementation. Currently the only expressions supported are
  -atime [+-]N
  -mtime [+-]N
  -newer path_to_file
+ -exec[dir] executable [args] [{{}}] [more args];
```

Comment:
I think there should be a space before the semicolon, emphasizing that it's a separate arg.